src/group_theory/free_group.lean

```diff
@@ -864,3 +867,88 @@ fintype.subtype (list.to_finset $ red.enum L₁) $
 end reduce
 
 end free_group
+
+
+/-- `is_free_group G` means that `G` isomorphic to a free group. -/
```

Comment:
Is there any reason you moved this out of the previous file? I think it was fine there, and this file is already weird long.

Reply:
My reasoning was that it’s such a thin wrapper layer on top of `free_group` that it doesn’t really deserve it’s own file any more (before it seemed conceptually more separate), and to make it discoverable.

But I can move it back, of course. Shall I?

Reply:
I think it's sensible to have it separate, simply to keep files short. It's certainly a thin layer over `free_group`, but that layer is separate from all the other results about free group.

It also makes the blame easier to follow if you leave it in the same file.

Reply:
> but that layer is separate from all the other results about free group.

… with the exception of the proof that a group that satisfies the universal property _is_ a free group. But maybe that’s even a good sign that it should be its own file.

I’ll move it back.